### PR TITLE
Lina Update

### PR DESCRIPTION
**Pull request by ShadowTsuki**
Fixed ellipses issue and edited translation for Lina.

### CODE REVIEW

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-14/translation.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about the teacher's experience with men? Oh, just experience with men? Maybe I've experienced women, too?
+﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about this Sensei's experience with men? My, are you alright with just my experience with men? You don't know if I might have some experience with women too, right?
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-14/translation.txt:1]: This one seems a bit awkward, still.  I tried reworking. Either is fine.
```suggestion
﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about Sensei's experience with men? Oh, are you sure I only have experience with men? Maybe I've been with women, too, right?

```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-16/translation.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-﻿「きみとのセックス...素敵だったわ」=Sex with you... was wonderful.
+﻿「きみとのセックス……素敵だったわ」=Sex with you... it was wonderful.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-16/translation.txt:1]: I don't think the `it` is needed. If you see the `...` as a pause it works better without the `it`

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-14/translation.txt (hunk 1 above)
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/-14/translation.txt:1]: This one seems a bit awkward, still.  I tried reworking. Either is fine.
```suggestion
﻿「先生の男性経験が知りたい？　あら、男性経験だけでいいの？　女性経験もあるかもしれないわよ？」=You want to know about Sensei's experience with men? Oh, are you sure I've only had men? Maybe I've been with women, too, right?

```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/2/translation.txt [hunk 3]
```diff
@@ -1,25 +1,25 @@
-﻿「ねぇ、きみ、ちょっと相談なんだけど...」=Hey, you, I'd like to ask you about something...
-「え？　先生が学生に相談するんですか？」=What? A teacher asking a student for advice?
-「たまにはそういうこともあるわよ。あ、でも正確には...相談じゃなくて頼み事...かも」=It happens sometimes. Oh, but really it's... not exactly for advice, but a favour... maybe
-「あ、また力仕事でしょうか？　だったらいいですよ。何でも言ってください」=Oh, is it hard work again? That's fine. Please ask away.
-「力仕事...うーん、力仕事...なのかしら？　確かにそう言えなくもないけど...」=Hard work... Hmm, is it hard work..? You could certainly say that...
-「...？　一体どういう頼みですか？」=..? What exactly are you asking me to do?
+「ねぇ、きみ、ちょっと相談なんだけど……」=Hey, you, I'd like to ask you about something...
+「え？　先生が学生に相談するんですか？」=Eh? Is Sensei asking a student for advice?
+「たまにはそういうこともあるわよ。あ、でも正確には……相談じゃなくて頼み事……かも」=There are times where that happens. Ah, but for this case... not exactly for advice, but a favour... maybe.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/2/translation.txt:3]:
```suggestion
「たまにはそういうこともあるわよ。あ、でも正確には……相談じゃなくて頼み事……かも」=There are times where that happens. Ah, but in this case... not exactly for advice, but a favour... maybe.

```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/5/translation.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
-﻿「やだもう……こんなに良くされたら、私、きみの虜になっちゃいそう」=Oh my... If you keep doing this to me, I think I'm going to fall in love with you.
-「毎日こんな感じで抱いてもらえたら……なんて、想像しちゃったじゃない」=If you could hold me like this every day... I can only imagine.
-「あ……そんなに落ち込まなくていいのよ？　男の子にはそういう時もあるんだから」=Don't be so downhearted. There are times when boys are like that.
+﻿「やだもう……こんなに良くされたら、私、きみの虜になっちゃいそう」=Oh my... If you keep doing this to me, I might just fall for you.
+「毎日こんな感じで抱いてもらえたら……なんて、想像しちゃったじゃない」=If you could hold me like this every day... Just kidding, I'm just imagining it.
+「あ……そんなに落ち込まなくていいのよ？　男の子にはそういう時もあるんだから」=Ah... You don't be so downhearted, you know? There are times when boys are like that.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/5/translation.txt:3]:
```suggestion
「あ……そんなに落ち込まなくていいのよ？　男の子にはそういう時もあるんだから」=Ah... You don't need to be so downhearted, you know? There are times when boys are like that.

```

or 

```suggestion
「あ……そんなに落ち込まなくていいのよ？　男の子にはそういう時もあるんだから」=Ah... You shouldn't be so downhearted, you know? There are times when boys are like that.

```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/voice_map/translation.txt [hunk 5]
```diff
@@ -1,17 +1,17 @@
-﻿「あら、私だってたまにはここにいるのよ？」=Oh my, you know I'm here sometimes right?
-「もしかして保健室に用事だった？　すぐに戻るから待っててね」=Did you have to go to the infirmary? I'll be right there.
-「職員室って意外とざわざわしてるでしょう？　落ち着かないわよね」=Isn't the staff room a bit crowded? It's very uncomfortable.
+﻿「あら、私だってたまにはここにいるのよ？」=Oh my, you know I'm sometimescome here, right?
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/00/voice_map/translation.txt:1]:
```suggestion
﻿「あら、私だってたまにはここにいるのよ？」=Oh my, you know I come here sometimes, right?

```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/12/6/translation.txt [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
-（ううん、何だか下半身に違和感が……？）=(Umm, what's this feeling in my lower body..?)
+（ううん、何だか下半身に違和感が……？）=(Umm, what's this feeling in my lower body...?)
 「…………」=............
 「って……え？　え？」=Wha... huh? Huh?
-「おはよ♡　ぐっすり寝てたわね」=Good morning, handsome♡ You looked so good sleeping.
-「あの、先生は何をして……？」=Wait, what are you doing there..?
-「ふふっ、お疲れみたいだから、先生がたっぷり尽くしてあげるわ」=Fufuu, You look tired, so teacher will take care of everything.
+「おはよ♡　ぐっすり寝てたわね」=Good morning ♡ You looked like you were sleeping soundly.
+「あの、先生は何をして……？」=Um, what are you doing, Sensei...?
+「ふふっ、お疲れみたいだから、先生がたっぷり尽くしてあげるわ」=Fufuu, you looked so tired, so sensei here will take care of everything.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c-1/12/6/translation.txt:7]:
```suggestion
「ふふっ、お疲れみたいだから、先生がたっぷり尽くしてあげるわ」=Fufuu, you looked so tired, so Sensei will take care of everything.

```

**Review by GeBo1 — COMMENTED** [review]
Looks, good. I saw a few typos and added a few suggestions. 

There some `Sensei here` or `this Sensei` that would probably read more naturally as just `Sensei`. I didn't flag them all.

**Review by GeBo1 — APPROVED** [review]

**Comment by ShadowTsuki** [comment]
Thanks! Made changes accordingly!

**Review by ManlyMarco — APPROVED** [review]

**Pull request merged**